.github/workflows/e2e_run_all.yaml

```diff
@@ -66,6 +66,24 @@ jobs:
       git_ref: ${{ inputs.git_ref }}
     secrets: inherit
 
+  podvm_mkosi_amd64:
```

Comment:
combine the two jobs in a matrix strategy?

Reply:
Which jobs? Do you mean `podvm_mkosi_amd64` and `podvm_mkosi_s390x`? Maybe I'm missing something clever, but as I've explained a few things people wanted the amd64 and s390x e2e flows to be completely decoupled, so we don't get delayed/blocked by specific issues in an architecture.